crates/youki/src/commands/update.rs

```diff
@@ -0,0 +1,26 @@
+use std::path::PathBuf;
+
+use crate::commands::create_cgroup_manager;
+use anyhow::Result;
+use libcgroups::{self, common::ControllerOpt};
+use liboci_cli::Update;
+use oci_spec::runtime::{LinuxPids, LinuxResources};
+
+pub fn update(args: Update, root_path: PathBuf) -> Result<()> {
+    let cmanager = create_cgroup_manager(root_path, &args.container_id)?;
+
+    let mut linux_res = LinuxResources::default();
+    if args.pids_limit.is_some() {
+        let mut pids = LinuxPids::default();
+        pids.set_limit(args.pids_limit.unwrap());
+        linux_res.set_pids(Some(pids));
+    }
```

Comment:
How about this? This will eliminate the need `unwrap()` :wink: 
```suggestion
    if let Some(new_pids_limit) = args.pids_limit {
        let mut pids = LinuxPids::default();
        pids.set_limit(new_pids_limit);
        linux_res.set_pids(Some(pids));
    }
```

Reply:
Nice catch! Fixed.